crates/uv/src/commands/reporters.rs

```diff
@@ -832,3 +832,32 @@ impl ColorDisplay for BuildableSource<'_> {
         }
     }
 }
+
+pub(crate) struct BinaryDownloadReporter {
+    reporter: ProgressReporter,
+}
+
+impl BinaryDownloadReporter {
+    /// Initialize a [`BinaryDownloadReporter`] for a single binary download.
+    pub(crate) fn single(printer: Printer) -> Self {
+        let multi_progress = MultiProgress::with_draw_target(printer.target());
+        let root = multi_progress.add(ProgressBar::with_draw_target(Some(1), printer.target()));
```

Comment:
What does the `Some(1)` here do?

Reply:
Yeah I.. don't really know much about what's going on here. We can open an issue to dig into it.